src/tree/updater_gpu_hist.cu

```diff
@@ -20,10 +20,15 @@
 #include "../common/device_helpers.cuh"
 #include "../common/hist_util.h"
 #include "../common/host_device_vector.h"
+#include "../common/sync.h"
 #include "../common/timer.h"
 #include "../common/span.h"
 #include "param.h"
 #include "updater_gpu_common.cuh"
+#include <unistd.h>
+
+//#define DBGPRINTF(...) printf(__VA_ARGS__)
+#define DBGPRINTF(...)
```

Comment:
Is this necessary?

Reply:
This is not necessary. Feel free to drop it.